.rubocop.yml

```diff
@@ -1,5 +1,5 @@
-Rails:
```

Comment:
The Rails cops are useful, so I'd keep them in.

Reply:
Yeah, just updated another change to fix that Rails issue https://github.com/theforeman/foreman_default_hostgroup/pull/62/commits/805f3da3dbd688bf9d34ae57e8b7e6e7281187d6